task/buildah-remote/0.2/buildah-remote.yaml

```diff
@@ -585,6 +605,7 @@ spec:
           -v "$BUILD_DIR/.docker/:/root/.docker:Z" \
           -v "$BUILD_DIR/results/:/tekton/results:Z" \
           -v "$BUILD_DIR/scripts:/scripts:Z" \
+          "$PRIVILEGED_NESTED_FLAG" \
```

Comment:
I think this variable expansion won't work as expected, it turns multiple flags into a single argument

An array would work better

```bash
PRIVILEGED_NESTED_FLAGS=(--privileged --mount --type=bind,...)
```

Reply:
Changed to array.